airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/jobs/registry.py

```diff
@@ -3,10 +3,10 @@
 registries_inclusive = AssetSelection.keys(
     "metadata_directory_report", "cloud_registry_from_metadata", "oss_registry_from_metadata"
 ).upstream()
-registry_reports_inclusive = AssetSelection.keys("connector_registry_location_html", "connector_registry_location_markdown").upstream()
+registry_reports_inclusive = AssetSelection.keys("connector_registry_report").upstream()
```

Comment:
Could you explain the purpose of this?

Reply:
So the `.upstream` method tells dagster to materialize
1. Any asset matching the key
2. Any assets it depends on in the DAG

without it the report generation would fail or get stale data as it wouldn't materialize the spec cache, or the registry file data